helm/ffc-demo-api-gateway/templates/developer-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ .Values.workstream }}-developer-1-admin-rolebinding
```

Comment:
The naming convention used here is I've used a naming convention `{user}-{role}-{resourcetype}` in case it is not immediately obvious.